rest/blip_sync.go

```diff
@@ -665,17 +706,23 @@ func isCompressible(filename string, meta map[string]interface{}) bool {
 	return true // be optimistic by default
 }
 
+func (bh *blipHandler) logEndpointEntry(profile, endpoint string) {
+	// Prof is short for "Profile", which is basically the blip message type
+	bh.LogTo("SyncMsg", "#%03d: Prof:%s %s User:%s", bh.serialNumber, profile, endpoint, bh.effectiveUsername)
```

Comment:
We've switched 'Prof' to 'Type' everywhere else, didn't we?

Reply:
Yep, thanks for catching